webapp/javaTest/java/com/linecorp/centraldogma/webapp/ShiroCentralDogmaTestServer.java

```diff
@@ -72,36 +65,16 @@ public static void main(String[] args) throws IOException {
                     users.put(USERNAME2, PASSWORD2);
                     return iniConfig;
                 }))
+                // Needs for enabling mTLS.
+                .port(443, SessionProtocol.HTTPS)
```

Comment:
Not sure if there is a way to disable HTTP and enable only HTTPS, but I was debugging why login isn't working from the HTTP page for a good 10 minutes.

Reply:
No problem. Let me add a new one. 😉